.github/ISSUE_TEMPLATE/icon-request.md

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
Want to add a link to this at the top of the icons story? Well, technically a link to https://github.com/ni/nimble/issues/new/choose along with instructions to pick "Icon Design Request".

Reply:
We might also mention it in https://github.com/ni/nimble/blob/main/packages/nimble-tokens/CONTRIBUTING.md#updating-icons but that's maybe less valuable